tools/toolchains/gcc.py

```diff
@@ -273,13 +273,13 @@ def __init__(self, target, options=None, notify=None, macros=None, silent=False,
         GCC.__init__(self, target, options, notify, macros, silent, TOOLCHAIN_PATHS['GCC_ARM'], extra_verbose=extra_verbose)
 
         # Use latest gcc nanolib
-        if "big-build" in self.options:
+        if "std-lib" in self.options:
```

Comment:
@theotherjimmy I recall this can be set up in the settings script file as ``BUILD_OPTIONS`` ?